src/components/program/tab-components/JourneyTab.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import MediaQuery from 'react-responsive';
+import { breakpoints } from '@edx/paragon';
+
+import { MainContent, Sidebar } from '../../layout';
+import { ProgramMainContent } from '../main-content';
+import { ProgramSidebar } from '../sidebar';
+
+
+function JourneyTab() {
+  return (
+    <div className="container py-5">
+      <div className="row">
+        <MainContent>
+          <ProgramMainContent />
+        </MainContent>
+        <MediaQuery minWidth={breakpoints.large.minWidth}>
+          {matches => matches && (
```

Comment:
Please let me know that where these matches are coming from?

Reply:
https://www.npmjs.com/package/react-responsive

you can see with components example here and see its usage. this matches the property we passed with window size and render components accordingly. matches is the property of MediaQuery and is not declared by us anywhere